samples/MusicStore.Spa/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
BTW, yes it does auto-generate one, but the environment variable is wrong because tooling hasn't updated yet.